src/main/java/org/minions/devfund/chelo/WhackAMole.java

```diff
@@ -0,0 +1,259 @@
+     * @param x coordinate
+     * @param y coordinate
+     */
+    void whack(int x, int y) {
+        if (moleGrid[x][y] == 'M') {
+            moleGrid[x][y] = 'W';
+            score++;
+            molesLeft--;
+            attemptsLeft--;
+        } else {
+            attemptsLeft--;
+        }
+    }
+
+    /**
+     * A method to print the grid without showing where the moles are.
+     * For every spot that has recorded a whacked mole, print out a W, or * otherwise.
+     */
+    void printGridToUser() {
+        for (int i = 0; i < moleGrid.length; i++) {
+            for (int j = 0; j < moleGrid.length; j++) {
+                if (moleGrid[i][j] == 'M') {
+                    System.out.print("[*]");
+                } else {
+                    System.out.print("[" + moleGrid[i][j] + "]");
+                }
+            }
+            System.out.println(" ");
+        }
+    }
+
+    /**
+     * A method to print the grid without showing where the moles are.
+     * For every spot that has recorded a whacked mole print out a W, or * otherwise.
+     */
+    void printGrid() {
+        for (int i = 0; i < moleGrid.length; i++) {
+            for (int j = 0; j < moleGrid.length; j++) {
+                System.out.print("[" + moleGrid[i][j] + "]");
+            }
+            System.out.println(" ");
+        }
+    }
+
+    /**\
+     * A method to return Attempts Left value.
+     * @return remaining attempts
+     */
+    int getAttemptsLeft() {
+        return attemptsLeft;
+    }
+
+    /**
+     * A method to return moles left value.
+     * @return remaining moles
+     */
+    int getMolesLeft() {
+        return  molesLeft;
+    }
+
+    /**
+     * A method to print the current status for.
+     * Score,
+     * Attempts Left,
+     * Moles Left.
+     */
+    public void printStatus() {
+        System.out.println("Score: " + this.score
+                + ", Attempts left: " + this.attemptsLeft
+                + ", Moles left: " + this.molesLeft);
+    }
+
+    /**
+     * Main method to run the program.
+     * @param args main method
+     */
+    public static void main(final String[] args) {
+        System.out.println("Enter number of attempts:");
+        int numAttempts = readUserInputOnlyIntegers();
+        System.out.println("Enter dimension for the grid:");
+        int dimensionBegin = readUserInputOnlyIntegers();
+
+        WhackAMole whackAmole = new WhackAMole(numAttempts, dimensionBegin);
+
+        System.out.println("Enter the amount of moles");
+        int amountMoles = readUserInputOnlyIntegers();
+        whackAmole.placeTheMoles(amountMoles, dimensionBegin);
+
+        // Printing the grid as first action to know where the moles were placed
+        whackAmole.printGrid();
+
+        int userInputX, userInputY;
+
+        // While the attempts left are greater than zero the user can submit coordinates
+        while (whackAmole.getAttemptsLeft() > 0) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS135)